osf_tests/test_node.py

```diff
@@ -2,6 +2,7 @@
 
 from django.utils import timezone
 from django.core.exceptions import ValidationError as DjangoValidationError
+from django.db.models import Q as DQ
 from modularodm import Q
```

Comment:
This overlaps with your models PR and there's quite a few MODM Qs in this file so I didn't want too many lines (by either django-ifying all of those Qs or importing them as MQ). I figured whichever one of us deals with the conflict last (probably me) can switch the DQ to Q?